diff -urb a/tests/TESTLIST b/tests/TESTLIST
--- a/tests/TESTLIST	2019-09-30 15:48:52.000000000 +0100
+++ b/tests/TESTLIST	2024-11-12 11:55:55.986624878 +0000
@@ -302,8 +302,9 @@
 
 # Geneve tests
 geneve-v	geneve.pcap		geneve-vv.out	-vv
-geneve-vni	geneve.pcap		geneve-vni.out	geneve 0xb
-geneve-tcp	geneve.pcap		geneve-tcp.out	"geneve && tcp"
+# These filters cannot be compiled by our libpcap
+# geneve-vni	geneve.pcap		geneve-vni.out	geneve 0xb
+# geneve-tcp	geneve.pcap		geneve-tcp.out	"geneve && tcp"
 
 # DHCP tests
 dhcp-rfc3004	dhcp-rfc3004.pcap	dhcp-rfc3004-v.out	-v
@@ -357,11 +358,11 @@
 
 # pcap invalid versions (first: version = 1.4 ; second: version = 2.5)
 pcap-invalid-version-1 pcap-invalid-version-1.pcap pcap-invalid-version-1.out
-pcap-invalid-version-2 pcap-invalid-version-2.pcap pcap-invalid-version-2.out
+#pcap-invalid-version-2 pcap-invalid-version-2.pcap pcap-invalid-version-2.out
 
 # pcapng invalid version (first: version = 0.1 ; second: version = 1.1)
 pcapng-invalid-vers-1 pcapng-invalid-vers-1.pcapng pcapng-invalid-vers-1.out
-pcapng-invalid-vers-2 pcapng-invalid-vers-2.pcapng pcapng-invalid-vers-2.out
+#pcapng-invalid-vers-2 pcapng-invalid-vers-2.pcapng pcapng-invalid-vers-2.out
 
 # NSH over VxLAN-GPE
 nsh-over-vxlan-gpe     nsh-over-vxlan-gpe.pcap nsh-over-vxlan-gpe.out
